pkgpanda/build/__init__.py

```diff
@@ -38,7 +38,7 @@ def __init__(self):
         self.container = str()
 
     def run(self, cmd):
-        docker = ["docker", "run"]
+        docker = ["docker", "run", "--rm"]
```

Comment:
I want to only clean this one up if the build was successful (It's handy for people to be able to re-enter the build container when things fail so they can experiment)